thaw/src/field/field.rs

```diff
@@ -16,6 +16,9 @@ pub fn Field(
     /// The orientation of the label relative to the field component.
     #[prop(optional, into)]
     orientation: MaybeSignal<FieldOrientation>,
+    //Is this input field required
```

Comment:
Use document comments(`///`), not regular comments(`//`).

Reply:
done